ConsumerApi/RelationshipChangesDbContextSeeder.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Backbone.BuildingBlocks.API.Extensions;
+using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.Persistence.BlobStorage;
+using Backbone.Modules.Relationships.Application.Infrastructure;
+using Backbone.Modules.Relationships.Domain.Entities;
+using Backbone.Modules.Relationships.Infrastructure.Persistence.Database;
+using Backbone.Modules.Tokens.Infrastructure.Persistence.Database;
+using Backbone.Modules.Tokens.Infrastructure.Persistence.Repository;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Options;
+// ReSharper disable EntityFramework.NPlusOne.IncompleteDataUsage
+// ReSharper disable EntityFramework.NPlusOne.IncompleteDataQuery
+
+namespace Backbone.ConsumerApi;
+
+public class RelationshipChangesDbContextSeeder : IDbSeeder<RelationshipsDbContext>
+{
+    private readonly IBlobStorage? _blobStorage;
+    private readonly string? _blobRootFolder;
+
+    public RelationshipChangesDbContextSeeder(IServiceProvider serviceProvider)
+    {
+        _blobStorage = serviceProvider.GetService<IBlobStorage>();
+        _blobRootFolder = serviceProvider.GetService<IOptions<BlobOptions>>()!.Value.RootFolder;
+    }
+
+    public async Task SeedAsync(RelationshipsDbContext context)
+    {
+        if (_blobRootFolder == null)
```

Comment:
Maybe we should also check if _blobStorage is null. This way we can get rid of the `!`s below.

```suggestion
        if (_blobRootFolder == null || _blobStorage == null)
```